src/Transition/CreateNewIDFUsingRulesV24_2_0.f90

```diff
@@ -394,7 +394,9 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
               OldFanVO(NumOldFanVO)%coeff5 = TRIM(IDFRecords(Num)%Numbers(12))             !- Coefficient5 x**4
               OldFanVO(NumOldFanVO)%inletAirNodeName = TRIM(IDFRecords(Num)%Alphas(4))
               OldFanVO(NumOldFanVO)%outletAirNodeName = TRIM(IDFRecords(Num)%Alphas(5))
-              OldFanVO(NumOldFanVO)%endUseSubCat = TRIM(IDFRecords(Num)%Alphas(6))
+              IF (lEN(IDFRecords(Num)%Alphas) .eq. 6) THEN
+                OldFanVO(NumOldFanVO)%endUseSubCat = TRIM(IDFRecords(Num)%Alphas(6))
+              ENDIF
```

Comment:
@mjwitte I was going to write that I was concerned about using the LEN of Alphas here.  But this is directly grabbing it from the IDFRecords array, so I guess it's going to be accurate.  This is fine with me if you see it working for you @lymereJ and @mjwitte doesn't see any red flags.

Reply:
I had a similar concern, but I think it's ok. I suppose at some point before release, we should run the full set of testfiles from 24.1 through transition and make sure they run without errors.